packages/table/src/quadrants/_quadrants.scss

```diff
@@ -45,13 +59,13 @@ $table-quadrant-z-index-top-left: $table-quadrant-z-index-left + 1;
   right: 0;
   z-index: $table-quadrant-z-index-top;
 
-  .bp-table-row-headers,
-  .bp-table-column-headers {
+  .bp-table-menu {
```

Comment:
The background colors in this file are just temporary. I'll revert them before merging this feature branch back into `master`.